epicure-admin/src/data/types/backEndData.types.tsx

```diff
@@ -0,0 +1,30 @@
+export type Chef = {
+  _id: string;
+  name: string;
+  image: string;
+  description: string;
+  isActive: boolean;
+  isCeffOfWeek: boolean;
+};
+
+export type Restaurant = {
+  _id: string;
+  name: string;
+  image: string;
+  chef: string;
+  isActive: boolean;
+  isPopular: boolean;
+  stars: number;
+  signatureDishId: string;
+};
+
+export type Dish = {
```

Comment:
name convention can be confusing,  if using type so the type of Chef  will be ChefType if using in interface use IChef. 

```suggestion
export type ChefType = {
  _id: string;
  name: string;
  image: string;
  description: string;
  isActive: boolean;
  isCeffOfWeek: boolean;
};

export type RestaurantType = {
  _id: string;
  name: string;
  image: string;
  chef: string;
  isActive: boolean;
  isPopular: boolean;
  stars: number;
  signatureDishId: string;
};

export type DishType = {
```